scalatest/src/main/scala/org/scalatest/refspec/RefSpecLike.scala

```diff
@@ -100,7 +100,7 @@ trait RefSpecLike extends TestSuite with Informing with Notifying with Alerting
         }
         
         def register(o: AnyRef): Unit = {
-          val testMethods = o.getClass.getMethods.filter(isTestMethod(_)).sorted(MethodNameEncodedOrdering)
+          val testMethods = o.getClass.getMethods.map(_.nn).filter(isTestMethod(_)).sorted(MethodNameEncodedOrdering)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.